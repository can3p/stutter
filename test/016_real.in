(list
 (+ 100 1.1 100)
 (+ 1.12345 100)
 (- 100 1.1)
 (- 100.0 40.232)
 (* 100 2.2)
 (* 2.2 100)
 (/ 100 1.1)
 (/ 200.5 222))
