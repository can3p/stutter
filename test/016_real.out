(201.100000 101.123450 98.900000 59.768000 220.000000 220.000000 90.909091 0.903153)
